aws/app.py

```diff
@@ -44,6 +54,17 @@ def __init__(self, app: core.App, id: str, **kwargs) -> None:
                 'LambdaRestApi',
                 handler=web_server_handler)
 
+        # S3 Bucket for Gateway to use
+        remove = core.RemovalPolicy.DESTROY
+        if config['prod']:
+            remove = core.RemovalPolicy.RETAIN
+        user_object_store = s3.Bucket(
+                self,
+                "LunarRocksPagesBucket",
+                bucket_name=config['pages_bucket'],
```

Comment:
We might run into S3 bucket name collisions if we assign a `bucket_name`. If we don't assign a `bucket_name`, CloudFormation will assign a name with "lunarrockspagesbucket" in it, plus a bunch of random characters that make it unique.

Reply:
Hmm. We definitely want to keep our prod s3 bucket naming specific -- it doesn't seem like we want to rely on the random suffix for that, as it won't be something we'll be standing up and taking down often. Also, using the random suffix for prod makes uploading the artifacts from github harder if not impossible because it would require keeping the action or secrets in sync with the latest random suffix. But maybe it would make sense to use the default random suffix in the case that `prod: false` in the stack config yaml, so if standing up the stack for development we wouldn't have to specify our own unique s3 bucket name

Reply:
Ah ok, I didn't think of the way this would work with GitHub actions. I think your suggestion makes sense for prod vs not prod. We can look for a good bucket name before doing the first prod deploy, and then stick with that.